src/test/java/com/ericsson/gerrit/plugins/eiffel/events/generators/EiffelEventGeneratorTest.java

```diff
@@ -212,8 +240,11 @@ private void verifyEiffelSourceChangeSubmittedEvent(
         assertEquals(errorMessage, COMMIT_ID, gitIdentifier.get("commitId").getAsString());
         assertEquals(errorMessage, BRANCH, gitIdentifier.get("branch").getAsString());
         assertEquals(errorMessage, PROJECT, gitIdentifier.get("repoName").getAsString());
+        assertEquals(errorMessage, "CHANGE", link1.get("type").getAsString());
+        assertEquals(errorMessage, "PREVIOUS_VERSION", link2.get("type").getAsString());
```

Comment:
There are 4 cases but you are only testing 1 case

Reply:
We are now covering: no links and link population in both events in the tests.